backend/local/local.go

```diff
@@ -1447,6 +1447,10 @@ func cleanRootPath(s string, noUNC bool, enc encoder.MultiEncoder) string {
 	if runtime.GOOS == "windows" {
 		s = filepath.ToSlash(s)
 		vol := filepath.VolumeName(s)
+		if vol == `\\?` {
+			// `\\?\C:`
+			vol = s[:6]
```

Comment:
That fix looks like it is an index error waiting to happen! 

Maybe wrap it in `if len(s) >= 6`?

Its a nasty fix - hopefully we can get rid of it soon!

Reply:
Will do! (And agreed.)

Reply:
Made this change.